{"name": "测试数据：【 wintest4，True，0，查询成功 】", "status": "failed", "statusDetails": {"message": "AssertionError: 查询用户 ==>> 接口返回码是 【 1004 】, 返回信息：查不到相关用户的信息 \nassert False == True\n +  where False = <core.result_base.ResultBase object at 0x103948f28>.success", "trace": "self = <testcases.api_test.test_01_get_user_info.TestGetUserInfo object at 0x103812f98>, username = 'wintest4', except_result = True\nexcept_code = 0, except_msg = '查询成功'\n\n    @allure.story(\"用例--获取某个用户信息\")\n    @allure.description(\"该用例是针对获取单个用户信息接口的测试\")\n    @allure.issue(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应BUG的链接地址\")\n    @allure.testcase(\"https://www.cnblogs.com/wintest\", name=\"点击，跳转到对应用例的链接地址\")\n    @allure.title(\"测试数据：【 {username}，{except_result}，{except_code}，{except_msg} 】\")\n    @pytest.mark.single\n    @pytest.mark.parametrize(\"username, except_result, except_code, except_msg\",\n                             api_data[\"test_get_get_one_user_info\"])\n    def test_get_get_one_user_info(self, username, except_result, except_code, except_msg):\n        logger.info(\"*************** 开始执行用例 ***************\")\n        step_2(username)\n        result = get_one_user_info(username)\n        # print(result.__dict__)\n        assert result.response.status_code == 200\n>       assert result.success == except_result, result.error\nE       AssertionError: 查询用户 ==>> 接口返回码是 【 1004 】, 返回信息：查不到相关用户的信息 \nE       assert False == True\nE        +  where False = <core.result_base.ResultBase object at 0x103948f28>.success\n\ntestcases/api_test/test_01_get_user_info.py:57: AssertionError"}, "description": "该用例是针对获取单个用户信息接口的测试", "steps": [{"name": "步骤1 ==>> 获取某个用户信息", "status": "passed", "parameters": [{"name": "username", "value": "'wintest4'"}], "start": 1632654815516, "stop": 1632654815516}], "attachments": [{"name": "log", "source": "c865563a-cd37-4cf9-9247-dbec79d764a9-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "9e9c14c3-2a4a-4768-933f-984add1f58b8-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "username", "value": "'wintest4'"}, {"name": "except_result", "value": "True"}, {"name": "except_code", "value": "0"}, {"name": "except_msg", "value": "'查询成功'"}], "start": 1632654815516, "stop": 1632654815526, "uuid": "432f6ce7-13cf-4b4b-8356-6e2fd3a4aaef", "historyId": "54eb6e4c952e7dabd97db254a2223ed6", "testCaseId": "11ffae3263ffa61feddcb26de6c12bb7", "fullName": "testcases.api_test.test_01_get_user_info.TestGetUserInfo#test_get_get_one_user_info", "labels": [{"name": "story", "value": "用例--获取某个用户信息"}, {"name": "severity", "value": "trivial"}, {"name": "feature", "value": "获取用户信息模块"}, {"name": "epic", "value": "针对单个接口的测试"}, {"name": "tag", "value": "single"}, {"name": "parentSuite", "value": "testcases.api_test"}, {"name": "suite", "value": "test_01_get_user_info"}, {"name": "subSuite", "value": "TestGetUserInfo"}, {"name": "host", "value": "txxdembp"}, {"name": "thread", "value": "11611-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcases.api_test.test_01_get_user_info"}], "links": [{"type": "test_case", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应用例的链接地址"}, {"type": "issue", "url": "https://www.cnblogs.com/wintest", "name": "点击，跳转到对应BUG的链接地址"}]}